### PR TITLE
Task/721 make contribution admin to see all items

### PR DESCRIPTION
**Pull request by ywkim312**
## Description
Added detail pages in catalog to show items separately based on user privilege. 

**Fixes #(add issue number here and remove parentheses)**

[comment]: # (This project only accepts pull requests related to open issues.)
[comment]: # (If suggesting a new feature or change, please discuss it in an issue first.)
[comment]: # (If fixing a bug, there should be an issue describing it with steps to reproduce.)

## Review Time Estimate
_Please give your idea of how soon this pull request needs to be reviewed by selecting one of the options below. This can be based on the criticality of the issue at hand and/or other relevant factors._

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Immediately
- [x] Within a week
- [ ] When possible

## Type of changes
_Please select a relevant option:_

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Bug fix (non-breaking change which fixes an issue).
- [x] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] Other (any another change that does not fall in one of the above categories.)

## Checklist:
_Please select all applicable options:_

[comment]: # (To select your options, please put an 'x' in the all boxes that apply.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [x] I have signed the Rokwire Contributor License Agreement ([CLA](https://rokwire.org/rokwire_cla)). (_Any contributor who is not an employee of the University of Illinois whose official duties include contributing to the Rokwire software, or who is not paid by the Rokwire project, needs to sign the CLA before their contribution can be accepted._)
- [x] I have updated the [CHANGELOG](../CHANGELOG.md).
- [x] I have read the [Contributor Guidelines](../CONTRIBUTING.md).
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [ ] My change requires updating the documentation.
- [ ] I have made necessary changes to the documentation.
- [ ] I have added tests related to my changes.
- [ ] My changes generate no new warnings.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

---

[comment]: # (Template credit: This pull request template is based on Embedded Artistry {https://github.com/embeddedartistry/templates/blob/master/.github/PULL_REQUEST_TEMPLATE.md}, Clowder {https://github.com/clowder-framework/clowder/blob/develop/.github/PULL_REQUEST_TEMPLATE.md}, and TalAter {https://github.com/TalAter/open-source-templates} templates.)

### CODE REVIEW

**Comment by sandeep-ps** [comment]
Please resolve the merge conflicts here. Thanks.

**Comment by ywkim312** [comment]
> Please resolve the merge conflicts here. Thanks.

resolved

File: CHANGELOG.md [hunk 1]
```diff
@@ -21,6 +21,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Add group authorization to PUT/events/{event_id}/images/{image_id} endpoint.[#719](https://github.com/rokwire/rokwire-building-blocks-api/issues/719)
 - Add group authorization to DELETE /events/{event_id}/images/{image_id} endpoint.[#720](https://github.com/rokwire/rokwire-building-blocks-api/issues/720)
 - Make catalog pages to show different items for reviewers and non reviewers. [#709](https://github.com/rokwire/rokwire-building-blocks-api/issues/709)
+- Make catalog pages to show all the items for contribution admins. [#721](https://github.com/rokwire/rokwire-building-blocks-api/issues/721)
```
Comment (sandeep-ps) [inline CHANGELOG.md:24]: There doesn't seem to be an issue with number 721. I think this needs to be updated.

File: CHANGELOG.md [hunk 2]
```diff
@@ -21,6 +21,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- Make contribution details pages to show all items for reviewers and admins. [#728](https://github.com/rokwire/rokwire-building-blocks-api/issues/728)
```
Comment (sandeep-ps) [inline CHANGELOG.md:25]: Please move all of the new CHANGELOG entries need to the Unreleased section. Thanks.

File: CHANGELOG.md (hunk 1 above)
Comment (ywkim312) [reply CHANGELOG.md]: changed it to 728

File: CHANGELOG.md (hunk 2 above)
Comment (ywkim312) [reply CHANGELOG.md]: Moved to unreleased... sorry

Comment (sandeep-ps) [reply CHANGELOG.md]: No issues. Thank you.

**Comment by sandeep-ps** [comment]
Please resolve the merge conflicts when you get a chance. Please put the most recent change at the end of the section. Thank you.

**Comment by ywkim312** [comment]
> Please resolve the merge conflicts here. Thanks.

resolved

**Review by sandeep-ps — APPROVED** [review]

**Pull request merged**